Rawl_B502/dend_rainn_eter_macco_ailella_flainn_bic.trig

```diff
@@ -104,31 +123,35 @@
     irishRel:nomName "Lóch Már" ;
     irishRel:ancestorOfGroup <#Éoganachta>;
     rel:childOf <#MaFemis>;
-    rdfs:comment "a quo sunt Éoganachta & Fianna Luigne Úi Dedaid Deocluaid".
-    #I am not sure how to interpret "Fianna Luigne Úi Dedaid Deocluaid m. Fiachach Oele" - CY
-    #It is ambiguous. Fiacha Oele could be the father of Dedad Deocluaid or the father of Ma Femis.
-    #And the idea of Fianna descending from someone as if they are a kin-group is interesting. EPT
+    rdfs:comment "a quo sunt Éoganachta & Fianna Luigne Úi Dedaid Deocluaid";
+    owl:sameAs <http://example.com/LL/senchas_síl_ébir.trig#Loich>.
+    # I am not sure how to interpret "Fianna Luigne Úi Dedaid Deocluaid m. Fiachach Oele" - CY
+    # It is ambiguous. Fiacha Oele could be the father of Dedad Deocluaid or the father of Ma Femis.
+    # And the idea of Fianna descending from someone as if they are a kin-group is interesting. EPT
+    # Update: Having finally located Ma Femis elsewhere in the genealogies, he does not seem to be
+    # the son of Fiacha Oele. EPT
```

Comment:
Should we leave this comment here or remove it with a fuller explanation?

Reply:
If you have a fuller explanation, please add it in. Otherwise, I think we should leave this as is until we understand it better.